### ISSUE TITLE
Instruction Typo in all pages `aurelia.plugin` should be `aurelia.use.plugin`

### ISSUE THREAD

**Issue by ghiscoding** [aurelia-toolbelt/aurelia-toolbelt#56]
#### instructions typo
Hello I just started using your lib and just wanted to mention that all the page instructions have the wrong code to import the plugin. For example [Modals](https://aurelia-toolbelt.github.io/#/bootstrap/modal) says to register the plugin through 
```javascript
aurelia.plugin(PLATFORM.moduleName('aurelia-toolbelt'));
``` 
while it should 
```javascript
aurelia.use.plugin(PLATFORM.moduleName('aurelia-toolbelt'));
```

Also in the [README](https://github.com/aurelia-toolbelt/aurelia-toolbelt#how-to-use-the-plugin) it says 
```javascript
aurelia.use(PLATFORM.moduleName('aurelia-toolbelt'));
```
which is again wrong but with different typo.

#### different subject
Out of subject, is that normal that the following doesn't work
```javascript
aurelia.use.plugin(PLATFORM.moduleName('aurelia-toolbelt/components/bootstrap/modal'));
```
I only get it to work with the entire bootstrap component
```javascript
aurelia.use.plugin(PLATFORM.moduleName('aurelia-toolbelt/components/bootstrap'));
```

**Comment by shahabganji** [comment]
Hi @ghiscoding 

About the main subject of this thread, you are right :see_no_evil: , I'll try to correct all of them ASAP.

The modal just worked fine for me, so I explain what I did and if that's not your case I kindly ask you to upload a sample repo so I can take a look. what I did is as following:

1. I created a new sample of Aurelia( since I'm a dotnet developer I used ```AureliaToolbelt.AspNetCore``` template )
```
dotnet new aut
```

2. change the ```configure``` method of ```main.ts```
```cs
...
aurelia.use
    .standardConfiguration()
    .plugin( PLATFORM.moduleName( 'aurelia-toolbelt/components/bootstrap/modal' ) )
    .feature(PLATFORM.moduleName('resources/index'));
...
```

3. change the content of the ```app.html```
```html
<template>

  <div class="container">

    <!-- you can use abt-button too, you should add that plugin in your main.ts file -->
    <!-- .plugin( PLATFORM.moduleName( 'aurelia-toolbelt/components/bootstrap/button' ) ) -->
    <button role="button" id="showModal"  type="button" data-toggle="modal" data-target="#a22c5596cacff4fc99452f1a659ea8c57">
        Launch demo modal
    </button>
    
    <!-- Modal -->
    <abt-modal open-by="showModal">
      <abt-modal-header>
        <abt-modal-title class="modal-title-aurelia">
          <h5>Aurelia Toolbelt Dialog</h5>
        </abt-modal-title>
      </abt-modal-header>
      <abt-modal-body>
        <div>
          Hooray, you see a message in a dialog.
        </div>
      </abt-modal-body>
      <abt-modal-footer>
        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
        <button type="button" class="btn btn-primary">Save changes</button>
      </abt-modal-footer>
    </abt-modal>

  </div>

</template>

```

**Comment by ghiscoding** [comment]
@shahabganji 
Thanks for your nice explanation, I actually realized why I thought it wasn't working. It actually does work but the button of modal demo was actually simply shown as just text (not the look of a button, however the text was clickable), so the modal demo also needs the `aurelia-toolbelt/components/bootstrap/button` for it to show the same as the demo. 

I don't want to make this simple issue become a long post of multiple subjects but I have another  question. I would like to use the Button Groups as radio button, to make the user choose 1 option out of 3 options, however in your demo I don't see how to bind the `checked` value.

Thanks for the reply

**Comment by shahabganji** [comment]
> so the modal demo also needs the aurelia-toolbelt/components/bootstrap/button for it to show the same as the demo.

yes exactly, since you just required the ```modal``` part of the toolbelt, thus other parts will be ignored in bundling and as a result,```abt-button``` tag is not recognizable by the framework.

> I would like to use the Button Groups as radio button,

[This](https://getbootstrap.com/docs/4.1/components/buttons/#checkbox-and-radio-buttons) feature of bootstrap is not implemented in toolbelt :see_no_evil:, because at the time of writing I had no idea whether it should be a mixed feature of ```abt-button``` or ```abt-button-group``` or another separate component. I would grateful if you come up with a suggestion and will be more than happy to implement your proposal.

**Comment by ghiscoding** [comment]
Will close original issue since it's already fixed and will continue the discussion in #57 

Thanks
